barcampcanterbury.com/index.html

```diff
@@ -150,31 +155,64 @@ <h2 id="activities">Activities</h2>
-				<p>This years theme is loosely around <a href="https://en.wikipedia.org/wiki/The_Fifth_Element">The Fifth Element</a></p>
-				<p>Our 4 event rooms will be titled Blaze, Cloud, Flood and Terra</p>
+				<p>This years theme is loosely around <a href="https://en.wikipedia.org/wiki/The_Fifth_Element">The Fifth Element</a>.</p>
+				<p>Our 4 event rooms will be titled Blaze, Cloud, Flood and Terra.</p>
 				<p>Attendees will be assigned one of the 4 elemental tribes and invited to partake in an optional treasure hunt and puzzles throughout the day.</p>
 				<p>There is an impending great calamity. What will the tribes do to protect the world?</p>
 			</div>
 			<div class="col-md">
-				<h3>Digital Badges</h3>
-				<p>The first 40 ticket signups will have a custom barcamp digital badge with an IR sensor and LED's to enhance the trasure-hunt.</p>
+				<h3>Talks</h3>
+                <p>On the day we will share a link to the session grid. If
+                you'd like to give a talk, you simply add the details of the
+                talk to the grid. It's as easy as that!</p>
+
+                <p>Need ideas for the topic of your talk? Here's <a
+                    href="https://twitter.com/barcampcant/status/1127676031656243200">the
+                    session grid for BarCamp London 2019</a>.</p>
+
+                <p>Talks can be about just about anything. Although talks tend
+                to be tech-related, there's no strict requirement for them to
+                be. Attendees don't <i>have</i> to give a talk, although we do
+                encourage everyone to get involved in some way.</p>
+			</div>
+		</div>
+		<h3>Digital Badges</h3>
+		<div class="row my-3">
+			<div class="col-md">
+				<p>The first 40 ticket signups will have a custom BarCamp digital badge with an IR sensor and LED's to enhance the treasure-hunt.</p>
 				<p>The badges will activate with different activities.</p>
-				<p>The badges and puzzles have been designed/engineerd/built by
+				<p>The badges and puzzles have been designed/engineered/built by
 					<a href="https://www.cs.kent.ac.uk/people/staff/dk246/">Daniel Knox</a>
 					and
 					<a href="https://www.cs.kent.ac.uk/people/staff/kg249/">Keith Greenhow</a>
 					from the
-					<a href="https://www.cs.kent.ac.uk/makerspace/index.html">School of Computing's Makerspace - The Shed</a>
+					<a href="https://www.cs.kent.ac.uk/makerspace/index.html">School of Computing's Makerspace - The Shed</a>.
 				</p>
 			</div>
-		</div>
-		<h3>Digital Badge WIP</h3>
-		<div class="row my-3">
 			<div class="col-sm" style="text-align: center;">
 				<img src="images/2019/badge_wip_1.jpg" class="img-thumbnail" style="max-height: 15rem;"/>
 				<img src="images/2019/badge_wip_2.jpg" class="img-thumbnail" style="max-height: 15rem;"/>
 			</div>
 		</div>
+		<div class="row my-3">
+			<div class="col-sm">
+				<h3>After Event</h3>
+                <p>After the talk sessions, we will go somewhere for drinks,

```

Comment:
More generic? Add sponsorship link?

"After the main event there will be a social event, more details will be announced closer to the date."